### PR TITLE
Development/processcontainers cleanup

### PR DESCRIPTION
**Pull request by HaseenaSainul**

### CODE REVIEW

File: ProcessContainers/CMakeLists.txt [hunk 1]
```diff
@@ -27,6 +27,8 @@ set(MODULE_NAME ${NAMESPACE}${PROJECT_NAME})
 
 message("Setup ${MODULE_NAME} v${PROJECT_VERSION}")
 
+set(PLUGIN_PROCESSCONTAINERS_AUTOSTART false CACHE STRING "Automatically start ProcessContainers plugin")
```
Comment (artur-gebicz-ml) [inline ProcessContainers/CMakeLists.txt:30]: Could you please quote values?

File: ProcessContainers/Containers.config [hunk 2]
```diff
@@ -1,2 +1,2 @@
-set (autostart false)
-set (outofprocess false) # Important - should be false
\ No newline at end of file
+set (autostart PLUGIN_PROCESSCONTAINERS_AUTOSTART)
```
Comment (artur-gebicz-ml) [inline ProcessContainers/Containers.config:1]: Missing $ in front of variable name

**Review by artur-gebicz-ml — CHANGES_REQUESTED** [review]

File: ProcessContainers/Containers.config (hunk 2 above)
Comment (HaseenaSainul) [reply ProcessContainers/Containers.config]: done

File: ProcessContainers/CMakeLists.txt (hunk 1 above)
Comment (HaseenaSainul) [reply ProcessContainers/CMakeLists.txt]: done

**Review by artur-gebicz-ml — APPROVED** [review]

**Pull request merged**